modules/openideal_statistics/src/Plugin/Block/OpenidealStatisticsIdeaStatisticsBlock.php

```diff
@@ -79,6 +85,27 @@ public function build($challenge = NULL) {
       ],
     ];
 
+    // Do not show if idea is not in below then expert review workflow phase.
+    if ($this->configuration['show_five_stars'] && $node->bundle() == 'idea' &&
+      !in_array($node->moderation_state->value,
+        [
+          'published',
+          'draft_approval',
+          'draft',
+        ]
+      )) {
+      $settings = [
```

Comment:
I think I've seen this multiple times, perhaps it's a good idea to move some pieces of code to a separate class.

Reply:
Done.